internal/services/kusto/kusto_cluster_resource.go

```diff
@@ -213,6 +251,17 @@ func resourceKustoCluster() *pluginsdk.Resource {
 				Default:  true,
 			},
 
+			"restrict_outbound_network_access": {
+				Type:     pluginsdk.TypeString,
+				Optional: true,
+				Default:  string(kusto.ClusterNetworkAccessFlagDisabled),
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(kusto.ClusterNetworkAccessFlagEnabled),
+						string(kusto.ClusterNetworkAccessFlagDisabled),
+					}, false),
```

Comment:
this should be a boolean?
```suggestion
				Type:     pluginsdk.TypeBool,
				Optional: true,
				Default:  false,
```